### PR TITLE
Yemibox' Branch

### PR DESCRIPTION
**Pull request by yemibox51**
Summarize your proposed change.
Mention the training team using @githubteacher.
Use the keywords closes immediately followed by your issue number (e.g. closes #3)to note which Issue the Pull Request is addressing. When you do this, the issue will be automatically closed when the pull request is merged.

### CODE REVIEW

**Comment by zalew16** [comment]
.diff

**Comment by zalew16** [comment]
start:worker_info

**Comment by j-collings** [comment]
@yemibox51 This looks great.

**Comment by zalew16** [comment]
2017-02-19 21:26 GMT+00:00 j-collings <notifications@github.com>:

**Comment by Medlinator** [comment]
**@yemibox51 This looks great.**

File: _pins/Yemibox.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: Yemibox
+latitude: 40.058324
+longitude: -74.405661
```
Comment (hectorsector) [inline _pins/Yemibox.json]: @yemibox51 you'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

**Comment by zalew16** [comment]
git checkout mistrzem 
git scalić --no-FF Yemibox-New-Jeresy 
origin master git Push

**Comment by brianamarie** [comment]
Hi @yemibox51! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@yemibox51 I'm going to merge now. Nice work!

**Pull request merged**